apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/CreateOrder.tsx

```diff
@@ -109,8 +184,7 @@ export const CreateOrder = () => {
             open={showLimitPriceWarning}
             onClose={() => setShowLimitPriceWarning(false)}
             onSubmit={() => {
-              swap()
-              setShowLimitPriceWarning(false)
```

Comment:
will it keep the same behaviour of displaying the limit price warning?

Reply:
Have added more logic related to this in `chore/sign-and-submit-tx` PR